### PR TITLE
Fix double airbomb spawn

### LINKED ISSUE #1223: Airstrikes _still_ double spawn.
They should drop 4 bombs, they do not. 

How many times does this need to be mentioned to get it fixed?

### PR DESCRIPTION
**Pull request by HakonRydland**
## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    airbomb was executed on hc as well when it spawned duplication the effect, now it will execute on either the firs hc or server

### Please specify which Issue this PR Resolves.
closes #1223
### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in singleplayer?
2. [x] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
call in an airstrike and watch, it also logs the airstrike heavely
********************************************************
Notes:

### CODE REVIEW

**Comment by jaj22** [comment]
Not seeing it. There's no multi-target execution here unless setWaypointStatements causes it, which would break about half of Barbolani's code. Possible (there is some other unexplained shit with setWaypointStatements), but you'd need to prove it.

**Comment by HakonRydland** [comment]
> Not seeing it. There's no multi-target execution here unless setWaypointStatements causes it, which would break about half of Barbolani's code. Possible (there is some other unexplained shit with setWaypointStatements), but you'd need to prove it.

image
from the logs on server with the extra logging stuff added

File: A3-Antistasi/functions/AI/fn_airbomb.sqf [hunk 1]
```diff
@@ -1,40 +1,44 @@
 if (not isServer and hasInterface) exitWith {};
 private _filename = "fn_airbomb";
-private ["_countX","_pilot","_typeX","_ammo","_cluster","_sleep","_bomb"];
-_pilot = _this select 0;
+[3, format ["Executing on: %1", clientOwner], _filename] call A3A_fnc_log;
+private ["_countX","_plane","_typeX","_ammo","_cluster","_sleep","_bomb"];
+_plane = vehicle (_this select 0);
 _typeX = _this select 1;
-_countX = 4;
+_countX = 6;
```
Comment (SonorousMeerkat) [inline A3-Antistasi/functions/AI/fn_airbomb.sqf:7]: Why was the number of bombs changed? I lowered it to 4 with airstrike sanity...

Comment (HakonRydland) [reply A3-Antistasi/functions/AI/fn_airbomb.sqf:7]: per discussion with bob, with the accuracy of the airstrike atm i dont think its a problem with two more bombs

Comment (Bob-Murphy) [reply A3-Antistasi/functions/AI/fn_airbomb.sqf:7]: Do 4, I forgot that I already talked about it with Meerkat.

Comment (HakonRydland) [reply A3-Antistasi/functions/AI/fn_airbomb.sqf]: done

**Pull request merged**